### PR TITLE
[swedish] sv_welcome_welcome-to-d

### PR DESCRIPTION
**Pull request by vladdeSV**

### CODE REVIEW

**Comment by vladdeSV** [comment]
`multi-paradigm (imperative, structured, object oriented, generic, functional programming purity, and even assembly)` is not yet translated. I have a hard time translating this.

**Comment by vladdeSV** [comment]
Ping @nordlow, could you act as a second second pair of eyes, checking for grammatical and spelling mistakes? There are also 4 other PR's by me.

**Comment by wilzbach** [comment]
Thanks a lot for your hard work @vladdeSV :)

> There are also 4 other PR's by me.

As Github forbids users to add labels, we use the prefix workaround. However you can still nicely filter for them:

https://github.com/stonemaster/dlang-tour/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Aopen%20%5Bswedish%5D

File: public/content/sv/welcome/welcome-to-d.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+och vi är glada i 'pull requests' som gör denna rundtur ännu bättre.
+
+## {SourceCode}
+
+```d
+import std.stdio;
+
+void main()
+{
+    writeln("Hello World!");
```
Comment (wilzbach) [inline public/content/sv/welcome/welcome-to-d.md]: Maybe you want to translate that too? ;-)

Comment (vladdeSV) [inline public/content/sv/welcome/welcome-to-d.md]: *Hello World!* is a general phrase, commonly used. It is also a very simple phrase.

Should those phrases be translated too?

Comment (wilzbach) [inline public/content/sv/welcome/welcome-to-d.md]: > Should those phrases be translated too?

tl;dr: It's your personal choice.

The programming books in German that I read in my youth always started with "Hallo Welt". However after skimming the different Wikipedia pages, I think there's a different cultural attitude if I am not mistake?

https://sv.wikipedia.org/wiki/Hello_World
https://de.wikipedia.org/wiki/Hallo-Welt-Programm

OT: This might be due to the phrase that I heard a couple of times "Swedish people speak better English than the English"

**Comment by vladdeSV** [comment]
Changed it nonetheless :)

Speaking of swedes knowing better English than the English, it is mostly people who spend a lot of time on the internet where the language is in English. However this topic is going overboard. There could a much larger discussion about this, but let us not have it here ;-)

File: public/content/sv/welcome/welcome-to-d.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Denna rundtur ger en överblick av detta __kraftfulla__ och __målande__
+språk vilket kompilerar direkt till __effektiv__, __ren__ maskin kod.
+
+{{/dmanmobile}}
+
+### Vad är D?
+
+D är höjdpunkten utav __decennier av erfarenhet inom att implementera kompilatorer__
+för många diverse språk och har en stort antal [unika funktioner](http://dlang.org/overview.html):
```
Comment (nordlow) [inline public/content/sv/welcome/welcome-to-d.md]: Replace `en` with `ett`.

File: public/content/sv/welcome/welcome-to-d.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+{{#dmandesktop}}
+
+- _hög nivå_ begrepp för fantastiskt modellerings kraft
+- _högt presterande_, kompilerat språk
+- statisk typning
+- evolutionen av C++ (utan dess misstag)
+- direkt gränssnitt till operativ systemets API:n och hårdvara
```
Comment (nordlow) [inline public/content/sv/welcome/welcome-to-d.md]: Togetherwrite `operativ systemets` to `operativsystemets`.

File: public/content/sv/welcome/welcome-to-d.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+- supersnabb kompilerings tid
+- tillåter minnes-säker programmering (SafeD)
```
Comment (nordlow) [inline public/content/sv/welcome/welcome-to-d.md]: Use `minnessäker` istället. Swedish doesn't separate words here, like english does.

File: public/content/sv/welcome/welcome-to-d.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+- _underhållbart_, _enkelt att förstå_ kod
+- kort inlärningskurva (C-likt syntax, likt Java och andra språk)
```
Comment (nordlow) [inline public/content/sv/welcome/welcome-to-d.md]: Use `C-lik` instead.

File: public/content/sv/welcome/welcome-to-d.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+- kompatibel med C applikationers binära gränssnitt
```
Comment (nordlow) [inline public/content/sv/welcome/welcome-to-d.md]: - Use `C-applikationers`.
- Add `(ABI-kompatibel)` at the end of the sentence.

File: public/content/sv/welcome/welcome-to-d.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+- multi-paradigm (imperative, structured, object oriented, generic, functional programming purity, and even assembly) //TODO: Translate to Swedish
+- inbyggt fel hinder (kontrakt, unitttest)
+
+... och många mer [funktioner](http://dlang.org/overview.html).
```
Comment (nordlow) [inline public/content/sv/welcome/welcome-to-d.md]: Use `många fler` istället.

File: public/content/sv/welcome/welcome-to-d.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+{{/dmandesktop}}
+
+### Om rundturen
+
+Varje sektion kommer med ett kod exempel som kan modifieras och användas med för
```
Comment (nordlow) [inline public/content/sv/welcome/welcome-to-d.md]: Use `användas till" istället för `användas med för`.

File: public/content/sv/welcome/welcome-to-d.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+att experimentera med Ds språk funktioner.
+Klicka på kör knappen (eller `Ctrl+enter`) för att kompilera och köra koden.
+
+### Bidra
+
+Denna rundtur är [öppen källkod](https://github.com/stonemaster/dlang-tour)
+och vi är glada i 'pull requests' som gör denna rundtur ännu bättre.
```
Comment (nordlow) [inline public/content/sv/welcome/welcome-to-d.md]: Replace with 

`och vi tar varmt emot 'pull requests' som gör denna rundtur ännu bättre.`

**Comment by nordlow** [comment]
Comments added!

**Comment by nordlow** [comment]
- imperative => imperativ
- structured => strukturerad
- object oriented => objekt-orienterad
- generic => generisk
- functional programming => funktionell programmering
- assembly => assembler

I don't `purity` can be translated. If I was forced I would be very verbose: `renhet, i betydelsen kontroll över funktioners sidoeffekter`

**Comment by vladdeSV** [comment]
Updated! Thanks @nordlow!

File: public/content/sv/welcome/welcome-to-d.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
+# Välkommen till D
+
+Välkommen till den interaktiva rundturen av *Programmings språket D*.
+
+{{#dmanmobile}}
+
+Denna rundtur ger en överblick av detta __kraftfulla__ och __målande__
```
Comment (Vild) [inline public/content/sv/welcome/welcome-to-d.md]: `målande` for me doesn't really feel like the right word.
I think something like `uttrycksfulla` sounds better.

**Comment by vladdeSV** [comment]
@Vild I agree with you, changed the word.

**Comment by wilzbach** [comment]
> Updated! Thanks @nordlow!
> @Vild I agree with you, changed the word.

@nordlow @Vild can we consider this translation done? :)

File: public/content/sv/welcome/welcome-to-d.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+- _högt presterande_, kompilerat språk
+- statisk typning
+- evolutionen av C++ (utan dess misstag)
+- direkt gränssnitt till operativsystemets API:n och hårdvara
+- supersnabb kompilerings tid
+- tillåter minnessäker programmering (SafeD)
+- _underhållbart_, _enkelt att förstå_ kod
+- kort inlärningskurva (C-lik syntax, likt Java och andra språk)
+- kompatibel med C-applikationers binära gränssnitt (ABI)
+- multi-paradigm (imperativ, strukturerad, objekt-orienterad, generisk, funktionell programmerings renhet (i betydelsen kontroll över funktioners sidoeffekter), och även assembler) //TODO: Translate to Swedish
```
Comment (Vild) [inline public/content/sv/welcome/welcome-to-d.md]: Only this left, I don't know what to write here but I'm marking it so you don't forget to fix it.

**Comment by vladdeSV** [comment]
Removed TODO, thanks @Vild for noticing

**Comment by wilzbach** [comment]
> Only this left

Thanks @vladdeSV @vild @nordlow :)
